graphql/authorizations.ts

```diff
@@ -344,6 +344,7 @@ export const authorizationEnhanceMap: Required<ResolversEnhanceMap> = {
     Achievement_template: allAdmin,
     User_achievement: allAdmin, // TODO change
     Achievement_event: allAdmin,
+    Job_run: { _all: nobody },
```

Comment:
why not admin only?

Reply:
Because this includes the permission for creating new entries in the Job Run Table via GraphQL, which we don't want anyone to do (also all resolvers / mutations are currently not added, so in that sense "nobody can do nothing")